{"additions": 1, "auther_ref": "IB/CMSSW_10_6_X/gcc700", "auther_sha": "73dee66847001a90bb544fe3c9811269260601bc", "author": "fabiocos", "body": "Addition of cms-externals/pythia8#14 by S. Mrenna fixing the issue cms-sw/cmssw#27918\r\n\r\nBackport of #5229", "branch": "IB/CMSSW_10_6_X/gcc700", "changed_files": 1, "comments": 7, "commits": 1, "created_at": "1568890391", "deletions": 1, "labels": ["comparison-available", "externals-pending", "orp-pending", "pending-signatures", "tests-approved"], "number": 5239, "release-notes": [], "review_comments": 0, "state": "open", "title": "[106X] Pythia8.240 fix for re-initialization", "updated_at": "1568914444", "user": "fabiocos"}